src/components/DayPickerRangeController.jsx

```diff
@@ -39,6 +39,10 @@ const propTypes = forbidExtraProps({
   startDate: momentPropTypes.momentObj,
   endDate: momentPropTypes.momentObj,
   onDatesChange: PropTypes.func,
+  range: PropTypes.shape({
```

Comment:
please store this propType somewhere reusable, so it can be shared wherever it's needed.

Reply:
👍